src/controllers/things_controller.ts

```diff
@@ -346,8 +346,8 @@ function build(): express.Router {
     const value = request.body;
     try {
       // Note: updatedValue may differ from value
```

Comment:
Side note: with the new changes this comment seems a little bit obscure. Should we keep it?

Reply:
Just noted that you actually removed this line in https://github.com/WebThingsIO/gateway/pull/2900/commits/1e76f8a1f9fb177eae7519bc076597772018bd5a#diff-b512196f53f6d1d852ec7d013d2070dce8f985f423299122a104947db6465551L348 .